bundles/org.openhab.voice.porcupineks/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="voice:porcupineks">
+		<parameter-group name="authentication">
+			<label>Pico Voice Credentials</label>
+		</parameter-group>
+		<parameter-group name="spot">
+			<label>Spot</label>
+		</parameter-group>
+		<parameter name="apiKey" type="text" required="true" groupName="authentication">
+			<label>Pico Voice API Key</label>
+			<description>API Key for using the underling binary.</description>
+		</parameter>
+		<parameter name="sensitivity" type="decimal" min="0" max="1" required="true" groupName="spot">
```

Comment:
As you provide a default value, you should not set it as required.
I have a doubt with "decimal" type, did you check that ?

Reply:
Yes decimal type seems to work fine. I can change the value, it is correctly accessed from the binding, and also the validations for min and max work fine on the ui.

Reply:
Verified: decimal is a valid value.